pallets/protos/src/lib.rs

```diff
@@ -143,6 +156,18 @@ pub mod pallet {
 	#[pallet::storage]
 	pub type UploadAuthorities<T: Config> = StorageValue<_, BTreeSet<ecdsa::Public>, ValueQuery>;
 
+	// Staking management
+	// (Amount staked, Last stake time)
+	#[pallet::storage]
+	pub type ProtoStakes<T: Config> = StorageDoubleMap<
```

Comment:
Would it be better to use a StorageNMap instead so that we can store the BlockNumber as another key?

Reply:
We don't really care though in this case, but for the Protos/RPCs maybe